scripts/kconfig/kconfigfunctions.py

```diff
@@ -335,6 +335,19 @@ def dt_compat_enabled(kconf, _, compat):
     return "n"
 
 
+def shields_list_contains(kconf, _, shield):
+    """
+    Returns "y" if cmake environment variable 'SHIELD_AS_LIST' exists and
+    'shield' is in after split()ing it, and "n" otherwise.
```

Comment:
`split()` splits on whitespace.

Reply:
I didn't get this comment..

Reply:
`split()`ing would split on whitespace rather than semicolons, so the docstring is wrong.

Could say something like `Return "y" if 'shield' appears in the comma-separated list in ... Returns "n" if ...`.

Reply:
@ulfalizer, gotcha.